sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/batching/SearchIndexingPublisher.java

```diff
@@ -143,11 +143,11 @@ public synchronized Duration getCurrentRetryDelay() {
                 this.actions.add(action);
             });
 
-        logger.verbose("Actions added, new pending queue size: {}.", this.actions.size());
+        LOGGER.verbose("Actions added, new pending queue size: {}.", this.actions.size());
 
         if (autoFlush && batchAvailableForProcessing()) {
             rescheduleFlush.run();
-            logger.verbose("Adding documents triggered batch size limit, sending documents for indexing.");
+            LOGGER.verbose("Adding documents triggered batch size limit, sending documents for indexing.");
```

Comment:
Again unrelated to this change but `verbose` logs should use the supplier API in ClientLogger.